src/System.Windows.Forms/src/System/Windows/Forms/ComboBox.ComboBoxChildNativeWindow.cs

```
@@ -41,9 +41,9 @@ protected override void WndProc(ref Message m)
                             object before = _owner.SelectedItem;
                             DefWndProc(ref m);
                             object after = _owner.SelectedItem;
-                            if (before != after)
+                            if (before != after && _owner.AccessibilityObject is ComboBoxAccessibleObject comboBoxAccessibleObject)
```

Can you add some comments around why we are silently failing on setting focus here for future reference?

---

Please check the comment